front/app/helpers/is-browser-outdated.js

```diff
@@ -0,0 +1,6 @@
+export default Ember.Helper.helper(function(){
+    if(!window.fetch){
```

Comment:
If we want to display a browser outdated message, maybe we should check for a feature other than fetch. Lack of fetch doesn't necessarily signify an outdated browser, see here: http://caniuse.com/#feat=fetch

Also we do already provide a polyfill with ember-fetch, which will provide this functionality on the browsers that do not yet support it. I think a better solution would possibly be to complete this issue here: https://github.com/dikaiosune/rust-dashboard/issues/108